src/Symfony/Bundle/FrameworkBundle/Command/DotenvDebugCommand.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Dotenv\Dotenv;
+
+/**
+ * A console command to debug current dotenv files with variables and values.
+ *
+ * @author Christopher Hertel <mail@christopher-hertel.de>
+ */
+final class DotenvDebugCommand extends Command
+{
+    protected static $defaultName = 'debug:dotenv';
+    protected static $defaultDescription = 'Lists all dotenv files with variables and values';
+
+    private $kernelEnvironment;
+    private $projectDirectory;
+
+    public function __construct(string $kernelEnvironment, string $projectDirectory)
+    {
+        $this->kernelEnvironment = $kernelEnvironment;
+        $this->projectDirectory = $projectDirectory;
+
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $io->title('Dotenv Variables & Files');
+
+        $envFiles = $this->getEnvFiles();
+        $availableFiles = array_filter($envFiles, function (string $file) {
+            return is_file($this->projectDirectory.\DIRECTORY_SEPARATOR.$file);
+        });
+
+        if (\in_array('.env.local.php', $availableFiles, true)) {
+            $io->warning('Due to existing dump file (.env.local.php) all other dotenv files are skipped.');
+        }
+
+        $io->section('Available Files');
```

Comment:
```suggestion
        $io->section('Available Files by descending priority');
```

to avoid adding more text rows on the output

Reply:
i dont think we should use Available Files, as more files can be available. IMHO these are the scanned files, see existing discussion: https://github.com/symfony/symfony/pull/42580#discussion_r689122714

if the suggestion is to reduce 1 line, im :+1: for `Scanned Files (in descending priority)`

Reply:
yes the idea is to reduce the arroudn text to focus on the legit output: the vars debuging :)